data/publiccloud/azure_vmss.sh

```diff
@@ -0,0 +1,150 @@
+#!/bin/bash -u
+set -o pipefail
+. $(dirname "${BASH_SOURCE[0]}")/azure_lib_fn.sh
+#create a virtual machine scale set
+#list, show and delete the created resource
+#Report list of cli's used
+
+# Virtual Machine Scale Set AZURE CLI testing script
+
+#Required AZ setting variable block
+grpname="${1:-oqaclitest}"
+location="${2:-westus}"
+vmname="${3:-oqaclivmss}"
+ssh_key="${4:-oqaclitest-sshkey}"
+vmximagename="${5:-UbuntuLTS}"
+
+# Variable block
+account="Azure RD"
+admin="azureuser"
+policy="automatic"
+inst_cnt=3
+runcmd="oqaclircss"
+
+# Resource creation
+echo "Set Azure Account { az account set }"
+cmd_status "az_account_set" az account set -n "$account"
+
+echo "Azure account set to $account"
+
+echo "Creating resource group { az group create }"
+cmd_status "az_group_create" az group create -n "$grpname" -l "$location" -o table
+echo "Created group $grpname"
+
+echo "Creating VM Scale Set { az vmss create }"
+cmd_status "az_vmss_create" az vmss create \
+    --resource-group "$grpname" \
+    --name "$vmname" \
+    --image "$vmximagename" \
+    --upgrade-policy-mode "$policy" \
+    --instance-count "$inst_cnt" \
+    --admin-username "$admin" \
+    --generate-ssh-keys
+echo "VM Scale set created"
+
+echo " List all the resource created by VM scale set $vmname { az resource list }"
+echo "***************************************************************************"
+cmd_status "az_resource_list" az resource list
```

Comment:
just in this example can you elaborate how it works 

```
cmd_status "az_resource_list" az resource list
az resource list --query "[?contains(name, '$vmname')].{name:name,type:type}"
```
I see "az resource list" in two lines , does it means that you calling it two times in a row ? why ?

Reply:
fixed it it was mistake. 
#List all the VMSS created resource
echo " List all the resource created by VM scale set ${vmname} { az resource list }"
echo "***************************************************************************"
cmd_status "az_resource_list" az resource list \
    --query "[?contains(name, '${vmname}')].{name:name,type:type}"